release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -256,6 +256,50 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 [id="ocp-4-14-storage"]
 === Storage
 
+[id="ocp-4-14-storage-standardized-sts-config"]
+==== Standardized STS configuration workflow
+{product-title} 4.14 provides a streamlined and standardized procedure to configure Security Token Service (STS) with the AWS Elastic File Storage (EFS) Container Storage Interface (CSI) Driver Operator.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#efs-sts_persistent-storage-csi-aws-efs[Configuring AWS EFS CSI Driver Operator with Security Token Service].
+
+[id="ocp-4-14-storage-rwop-access-mode"]
+==== Read Write Once Pod access mode (Technology Preview)
+{product-title} 4.14 introduces a new access mode for persistent volumes (PVs) and persistent volume claims (PVCs) called ReadWriteOncePod (RWOP), which can be used only in a single pod on a single node. This is compared to the existing ReadWriteOnce access mode where a PV or PVC can be used on a single node by many pods. This feature is supported with Technology Preview status.
+
+For more information, see xref:../storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[Access modes].
+
+[id="ocp-4-14-storage-gcp-filestore-csi-driver"]
+==== GCP Filestore storage CSI Driver Operator is generally available
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Compute Platform (GCP) Filestore Storage. The GCP Filestore CSI Driver Operator was introduced in {product-title} 4.12 with Technology Preview support. The GCP Filestore CSI Driver Operator is now generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc[Google Compute Platform Filestore CSI Driver Operator].
+
+[id="ocp-4-14-storage-automigration-vsphere"]
+==== Automatic CSI migration for VMware vSphere
```

Comment:
@lpettyjo We need to add the vsphere version and OCP upgrades consideration here

Reply:
Done.